src/excmds.ts

```diff
@@ -1309,6 +1328,7 @@ export async function tabopen(...addressarr: string[]) {
 
     activeTabContainerId().then(containerId => {
         if (containerId && config.get("tabopencontaineraware") === "true") openInNewTab(url, { active: active, cookieStoreId: containerId })
+        else if (container) openInNewTab(url, { active: active, cookieStoreId: "firefox-container-" + container })
```

Comment:
This should take priority over the line above it; `tabopen -c blah` should always open a tab in blah.

Reply:
Nice catch, I'll fix that real quick.